### DISCUSSION TITLE
Account not found

### DISCUSSION THREAD

**Discussion by rudra016** [community/community#81188] [New to GitHub]
when i open my github account link i.e https://github.com/rudra016, it shows nothing found, but my account is working fine, also the readme.md file also vanished from my profile.

pls help

**Comment by firasbelhiba** [comment]
me too I have the same issue , How did you solve it ?

**Comment by queenofcorgis** [comment]
Hi there @rudra016, 

Confirming I can see your profile 😃 

Were you able to fix this yourself or did this work itself out on its own?

**Reply by rudra016** [reply]
GitHub support helped me, basically they told me this bug happens a lot when some profiles are mistakenly assumed to be spam.

**Reply by Saivaraprasad48** [reply]
Hi @rudra016 I got the same issue here, Can you help me out, how can I resolve this means where can I find GitHub Support :)

**Reply by Saivaraprasad48** [reply]
Hi @queenofcorgis, I was facing this issue, can you confirm my profile too, Thank you in advance!
